### ISSUE TITLE
Add HEAD branch in `BRANCHES`

### ISSUE THREAD

**Issue by nikitabobko** [sublimehq/sublime_merge#471]
Here in `BRANCHES` section I'd like to see `HEAD` branch:
![image](https://user-images.githubusercontent.com/20517828/56606090-2a7e3400-660e-11e9-96be-75a324bb06ac.png)
which represents current branch.

**Comment by srbs** [comment]
But `HEAD` isn't a branch.. maybe it could go above the `BRANCHES` section & be a location without any children.

**Comment by nikitabobko** [comment]
@srbs thank you! Your suggestion is even better than my own. For many people I know them use "HEAD branch" very often in other git GUIs and placing it on top level along with BRANCHES is good idea.

**Comment by srbs** [comment]
@nikitabobko, feel free to modify your title & description if you think my suggested is the route to go. 😃

**Comment by vladimir-bebeshko** [comment]
I think it would be very convenient to have `HEAD` in Locations! I wonder why it's not there from the very beginning... This feature is the first in my wait-list.

**Comment by stdedos** [comment]
I usually "ride" with locations hidden.

Some mocked _Parent: `sha-of-HEAD`_ attribute could be added, above the commit message prompt (like this prompt above "commited" commits)
![image](https://user-images.githubusercontent.com/133706/74830379-b0950500-531b-11ea-8fbd-f8288693af33.png)

**Comment by nikitabobko** [comment]
> I usually "ride" with locations hidden.

It's because it's designed really badly. See: #479

> Some mocked Parent: sha-of-HEAD attribute could be added

Well, not sure that it's good design solution. For me it'd look strange

**Comment by stdedos** [comment]
> Well, not sure that it's good design solution. For me it'd look strange

I don't think it would look as bad as you may imagine. If I pretend to be a mock designer:
![image](https://user-images.githubusercontent.com/133706/74839855-ac261780-532e-11ea-916a-5a1583ea227d.png)

That doesn't look half bad to me, it is UI-consistent, and gets the job done

**Comment by nikitabobko** [comment]
Forget what I said. It looks ok. It would be nice to have both:
1. Item in `Locations` side bar
2. Your suggested UI
